src/core/libmaven/mavenparameters.cpp

```diff
@@ -232,8 +232,17 @@ void MavenParameters::setOptionsDialogSettings(const char* key, const char* valu
 {
     mavenSettings[const_cast<char*>(key)] = const_cast<char*>(value);
 
-    if(strcmp(key, "ionizationMode") == 0)
-        ionizationMode = atof(value);
+    if(strcmp(key, "ionizationMode") == 0){
+        int indexOfIonizationMode=atoi(value);
+        /**
+         * map index of ionization-mode to value (-1,0,1) of ionizationMode
+         */
+        if(indexOfIonizationMode==1) ionizationMode=0;
```

Comment:
@IpankajI Shouldn't the indices be 0,1 and 2?

Reply:
@shubhra-agrawal 0 is for Auto-Detect, rest is tested.

Reply:
@IpankajI I was confusing this drop-down with the one on main window. My bad.